website/pages/api-docs/system/storage/raftautosnapshots.mdx

```diff
@@ -0,0 +1,247 @@
+---
+layout: api
+page_title: /sys/storage/raft/snapshot-auto - HTTP API
+sidebar_title: <code>/sys/storage/raft/snapshot-auto</code>
+description: |-
+
+  The `/sys/storage/raft/snapshot-auto` endpoints are used to manage automated
+  snapshots with Vault's Raft storage backend.
+
+  This is an Enterprise-only feature.
+---
+
+## Create/update an automated snapshots config
+
+**This endpoint requires sudo capability.**
+
+This endpoint creates or updates a named configuration.  Each configuration
+has an interval controlling how often snapshots are taken, a destination to
+which to write the snapshots, as well as a retention policy governing when
```

Comment:
The destination bit reads a little awkwardly to me. I might suggest something like: `a destination where the snapshots are written`

Reply:
Agreed